src/pages/_app.tsx

```diff
@@ -17,7 +21,78 @@ const MyApp: FunctionComponent<AppProps> = ({
     GOVUKFrontend.initAll();
   }, []);
 
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Head>
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, viewport-fit=cover"
+        />
+        <meta charSet="utf-8" />
+        <meta name="theme-color" content="#0b0c0c" />
+
+        <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+
+        <link
+          rel="shortcut icon"
+          sizes="16x16 32x32 48x48"
+          href="/assets/images/favicon.ico"
+          type="image/x-icon"
+        />
+        <link
+          rel="mask-icon"
+          href="/assets/images/govuk-mask-icon.svg"
+          color="#0b0c0c"
+        />
+        <link
+          rel="apple-touch-icon"
+          sizes="180x180"
+          href="/assets/images/govuk-apple-touch-icon-180x180.png"
+        />
+        <link
+          rel="apple-touch-icon"
+          sizes="167x167"
+          href="/assets/images/govuk-apple-touch-icon-167x167.png"
+        />
+        <link
+          rel="apple-touch-icon"
+          sizes="152x152"
+          href="/assets/images/govuk-apple-touch-icon-152x152.png"
+        />
+        <link
+          rel="apple-touch-icon"
+          href="/assets/images/govuk-apple-touch-icon.png"
+        />
+
+        <meta
+          property="og:image"
+          content="/assets/images/govuk-opengraph-image.png"
+        />
+      </Head>
+      <a href="#main-content" className="govuk-skip-link" role="main">
+        Skip to main content
+      </a>
+      <Header serviceName={"Beacon registration service"} homeLink={"#"} />
+      <PhaseBanner
+        phase={"BETA"}
+        bannerHtml={
```

Comment:
Would this not be better to pass in another React component to render? You could look to define this as a small functional component in this file if it's only used here?

Reply:
Going to change `<PhaseBanner>` component so that `bannerHtml` is replaced by child:

```jsx
<PhaseBanner phase={"BETA"} >
          <>
            This is a new MCA Show and Tell on 29 January 2021 – your{" "}
            <a className="govuk-link" href="#">
              feedback
            </a>{" "}
            will help us to improve it.
          </>
      </PhaseBanner>
```